text/0121-draft-environment-variable-conventions.md

```diff
@@ -0,0 +1,46 @@
+- Start Date: 2023-11-07
+- RFC Type: decision
+- RFC PR: https://github.com/getsentry/rfcs/pull/121
+- RFC Status: draft
+
+# Summary
+
+We already expose certain environment variables that are consumed by SDKs, such as `SENTRY_DSN` and `SENTRY_RELEASE`.
+To further ease usage, we should expose more options as such.
+This RFC aims to find a set of conventions mainly aimed for server-side SDKs.
+
+# Motivation
+
+Changing the SDK configuration most often requires a re-deployment of the application. This can be cumbersome for small changes,
+such as updating the `traces_sample_rate`. Given the rise in populairty of server-less or containerized deployments, where
+most configuration hails from envrionment variables, we should strive to make this workflow less cumbersome for our users.
+
+# Background
+
+The Laravel SDK exposes the majority of its configuration as environment variables, making it a breeze to update the configuration of the SDK easily.
+For a complete list of exposed variables, see https://github.com/getsentry/sentry-laravel/blob/9624a88c9cd9a50c22443fcdf3a0f77634b11210/config/sentry.php
+
+# Options Considered
+
+A first list of environment variables that all server sides SDKs should support could look like:
+
+- `SENTRY_DSN`
+- `SENTRY_RELEASE`
+- `SENTRY_ENVIRONMENT`
+- `SENTRY_SAMPLE_RATE`
+- `SENTRY_TRACES_SAMPLE_RATE`
+- `SENTRY_PROFILES_SAMPLE_RATE`
+- `SENTRY_DEBUG`
```

Comment:
Just stumbled across this RFC. We could also list `SENTRY_TRACE` and `SENTRY_BAGGAGE`, which could be picked up to continue traces.

Reply:
Yep, there is/was another RFC for that, but I can add it and actually move this RFC forward :grimacing: